package.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "pms",
+  "version": "1.0.0",
+  "description": "This is a product management system application built using TypeScript. To build it, I used ParcelJS.",
+  "type": "module",
+  "scripts": {
+    "dev": "parcel serve ./src/index.html --port 3000",
+    "type-check": "tsc -b --noEmit",
+    "lint": "eslint src --ext .ts",
+    "build": "npm run type-check && npm run lint && parcel build ./src/index.html --no-autoinstall --no-optimize",
+    "format": "prettier src --write"
```

It is a good idea to first run the format before running the lint. We can potentially avoid the conflicts if any.
```suggestion
    "format": "prettier src --write",
    "build": "npm run format && npm run type-check && npm run lint && parcel build ./src/index.html --no-autoinstall --no-optimize",
```

---

https://github.com/dhruvdhaduk-simform/product-management-ts/pull/1#discussion_r2016495428

i also have vs code setup for prettier.

---

Got it.